### PR TITLE
Update winds to 3.1.4

### PR DESCRIPTION
**Pull request by MartinForReal**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/winds.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'winds' do
-  version '2.1.174'
-  sha256 '53a316dc6a229b290eadc088376925d88f59777f73c8d153f74b282c8a41d906'
+  version '3.1.4'
+  sha256 'ebb116a2a3b7396eff1edfce7d69da31ea6f6d66efc04a98f6c260319eda2ba8'
 
   # s3.amazonaws.com was verified as official when first introduced to the cask
-  url "https://s3.amazonaws.com/winds-#{version.major}.0-releases/releases/Winds-#{version}.dmg"
+  url 'https://s3.amazonaws.com/winds-2.0-releases/releases/Winds-3.1.4.dmg'
```
Comment (ran-dall) [inline Casks/winds.rb]: @MartinForReal The URL is correct. Please revert it back. Should be ok.👌

**Review by ran-dall — CHANGES_REQUESTED** [review]
@MartinForReal The URL that initially was on the cask was correct. You don't' need to update it. 👍

File: Casks/winds.rb (hunk 1 above)
Comment (vitorgalvao) [reply Casks/winds.rb]: @ran-dall Technically it wasn’t, because the `winds-2.0-releases` part remained at `2`.

**Review by vitorgalvao — APPROVED** [review]

File: Casks/winds.rb (hunk 1 above)
Comment (ran-dall) [reply Casks/winds.rb]: @vitorgalvao Yes, this would have remained at 2. You are correct. I looking at the `#{version}`.

**Pull request merged**